test/data/repositories/rhel-9.5.json

```diff
@@ -0,0 +1,146 @@
+      "image_type_tags": [
+        "azure-rhui"
+      ]
+    },
+    {
+      "name": "google-compute-engine",
+      "baseurl": "https://rpmrepo.osbuild.org/v2/mirror/public/el9/el9-x86_64-google-compute-engine-20240515/",
+      "check_gpg": false,
+      "image_type_tags": [
+        "gce",
+        "gce-rhui"
+      ]
+    },
+    {
+      "name": "google-cloud-sdk",
+      "baseurl": "https://rpmrepo.osbuild.org/v2/mirror/public/el9/el9-x86_64-google-cloud-sdk-20240515/",
+      "check_gpg": false,
+      "image_type_tags": [
+        "gce",
+        "gce-rhui"
+      ]
+    }
+  ]
+}
```

Comment:
Mini-nitpick: trailing newline.

Reply:
I just copied the `rhel-9.4.json` and replaced `9.4` with `9.5` :innocent:

Reply:
I get this a lot when I edit the files manually too.  Schutzbot uses `json.dump()` which doesn't add a newline, but then most editors will add it when saved, so it's a constant battle.